CMakeLists.txt

```diff
@@ -328,6 +328,12 @@ include_directories(${Boost_INCLUDE_DIRS})
 
 find_package(LLVM REQUIRED CONFIG)
 message(STATUS "Found LLVM ${LLVM_PACKAGE_VERSION}")
+if(NOT DEFINED LLVM_ENABLE_RTTI)
+  MESSAGE(FATAL "Could not determine if LLVM has RTTI enabled.")
+endif()
+if(NOT ${LLVM_ENABLE_RTTI})
+  MESSAGE(FATAL "Galois requires a build of LLVM that includes RTTI. Most package managers do this already, but if you built LLVM from source you need to configure it with `-DLLVM_ENABLE_RTTI=ON`")
```

Comment:
I think it is `message(FATAL_ERROR)` https://cmake.org/cmake/help/latest/command/message.html

Reply:
Yep. You're totally right. Thanks for catching that. Updated.